.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -5,7 +5,7 @@
 jobs:
 - job: win
   pool:
-    vmImage: windows-2022
+    vmImage: windows-2019
```

Comment:
What's the reason for requiring `windows-2019`? We can override this in a way that survives the rerender proces, but manually editing this file won't work...

Reply:
manually editing this file has worked here, #35 #32 and #27. msvc default I believe

Reply:
> manually editing this file has worked here

Does that means it fails with the 2022 image? It's not ideal to do manually because you have to do it every time; if indeed it is necessary, we can set the config in a way that the rerender will not overwrite this every time.

> msvc default I believe

I don't know what you mean by that, could you explain? In any case we're using the 2022 images all throughout conda-forge, even though we're targeting the vs2019 toolset. That generally works without issues, so I'm curious what's happening here.

Reply:
Ping @timkpaine